### ISSUE TITLE
Failure to build with nightly

### ISSUE THREAD

**Issue by kripken** [bytecodealliance/wizer#38]
Following the build instructions, I did
```rust
cargo install wizer --all-features
```
But I get a bunch of errors like this:
```rust
error: cannot find macro `asm` in this scope
  --> /home/alon/.cargo/registry/src/github.com-1ecc6299db9ec823/rsix-0.25.1/src/imp/linux_raw/arch/inline/x86_64.rs:57:5
   |
57 |     asm!(
   |     ^^^
   |
   = note: consider importing one of these items:
           std::arch::asm
           core::arch::asm
[..]
error: could not compile `rsix` due to 14 previous errors
warning: build failed, waiting for other jobs to finish...
error: failed to compile `wizer v1.3.4`, intermediate artifacts can be found at `/tmp/cargo-installjaUMur`
```
That is using
```rust
$ cargo --version
cargo 1.59.0-nightly (a359ce160 2021-12-14)
```
I'm not very familiar with cargo errors, sorry if this is the wrong place to report the issue (should I report it on that `rsix` package?)

**Comment by fitzgen** [comment]
Thanks for filing a bug!

I think this is the same upstream issue that is also being dealt with [here](https://github.com/bytecodealliance/wasm-tools/pull/428). I'll update the `wasmtime` dep (which transitively depends on `rustix` ne `rsix`) once the fix is merged.

**Comment by conradgrobler** [comment]
It seems that the fix has been merged in `rustix` (https://github.com/bytecodealliance/rustix/pull/143). Would it be possible to release a new version that depends on `wasmtime` v 0.32.1 or later?

**Comment by conradgrobler** [comment]
Sorry, my previous comment was premature. It seems that the version of `cap-std` must be updated to support building with recent nightly rust. I created #47 to do update `cap-std`.

**Comment by fitzgen** [comment]
Fixed in #47
